src/signals/my-incidents/types.ts

```diff
@@ -1,18 +1,20 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2022 - 2023 Gemeente Amsterdam
 import type { Item } from 'shared/types/extraProperties'
+import type { LegacyItems } from 'shared/types/extraProperties'
 import type Location from 'types/location'
 export interface MyIncidentsValue {
-  email?: string
   setEmail: (email: string) => void
-  incidentsList?: MyIncident[]
   setIncidentsList: (incidentsList: MyIncident[]) => void
+  incidentsList?: MyIncident[]
+  email?: string
 }
 
 interface Attachment {
```

Comment:
Isn't this also a case of "There already is an Attachment type. Extract that instead"? Incidentdetail of incident-management also has its own type of Attachment.